SR2020/2019-10-21-kit-disclaimer-form.md

```diff
@@ -0,0 +1,20 @@
+---
+to: sr2020-teams
+subject: Kit Disclaimer Forms
+attachments:
+  - https://drive.google.com/a/studentrobotics.org/file/d/1aCHnZ6o8DkLzHef1C2lz_iIkZkOjYP6P/view?usp=sharing
+---
+
+Hi all,
+
+Kickstart is less than a week away, and we're putting the finishing touches to kickstart. If you're attending a kickstart event, your kits are ready and waiting at the venue. For those who can't attend kickstart, we're currently working out the details for how to ship you your kit.
```

Comment:
Perhaps for the shipped kits say something like "we'll contact you shortly to confirm details of shipping your kits"? This avoids it sounding like it's something we've not yet got sorted out.